### ISSUE TITLE
Name Backdrop -fanart and not banner

### ISSUE THREAD

**Issue by euphoria360** [Radarr/Radarr#2556]
**Description:**

This issue is a followup of Issue #167  (Merged Pullup #2506 ).
The problem is that downloaded artworks by **Kodi (XBMC) / Emby Metadata Downloader** are not named correctly based on [default Kodi naming scheme](https://kodi.wiki/view/Movie_artwork)

Currently, the downloaded artworks are named like this in each movie's folder:
NFO File: `<NameOfVideoFile>.nfo`
Poster: `<NameOfVideoFile>-poster.jpg`
Fanart: `<NameOfVideoFile>-banner.jpg` (wrong)

But, based on Kodi Wiki and by assuming single movie in each directory, the naming should be like this:
NFO File: `<NameOfVideoFile>.nfo`
Poster: `<NameOfVideoFile>-poster.jpg`
Fanart: `<NameOfVideoFile>-fanart.jpg`
Actor Image: `.actors/<FirstName>_<LastName>.jpg`

**Radarr Version:**
0.2.0.954

**Mono Version:**
4.8.1 (Stable 4.8.1.0/22a39d7 Tue May 2 23:55:41 UTC 2017)

**Debug Logs:**
Not needed

**Comment by galli-leo** [comment]
But the banner file isn't fanart, it's a banner? Also we don't have actor's images, so not sure why you included that.

**Comment by euphoria360** [comment]
The file downloaded as `<NameOfVideoFile>-banner.jpg` is actually a fanart file and should be named `<NameOfVideoFile>-fanart.jpg`. and despite these 2 pictures, no other artwork file is being downloaded.

**Comment by euphoria360** [comment]
The reason I added them is that the 3 types of artwork mentioned above are part of basic movie artwork of kodi. But I guess including actor images are another matter, probably better discussed in #2557 .

**Comment by galli-leo** [comment]
Problem is, we don't know that from our metadata provider. TMDB just says backdrop.

**Comment by euphoria360** [comment]
Well, Backdrop is another name for fanart a 16:9 image preferably 1920x1080.
Banners are totally different matter, a ribbon like image with 1:0185 aspect ratio and preferably 1000x185.

These 2 artworks are created for totally different views, fanart or backdrop is a landscape file always paired with poster.
Banners are used more in tv show listing views, where you can see each tv show like a ribbon.

It would be nice to take a closer look to Kodi Wiki link I provided and look at the screenshots provided.

**Comment by AnthraxLeprosyPi** [comment]
Hey guys, 

is this going to be fixed or further discussed in hte referenced issue?
It would be nice to have the TMDB backdrop named -fanart and not -banner as I'll bet this will get confusing for others as well real fast ;-)

keep up your marvelous work.

best regards
Anthrax

**Comment by euphoria360** [comment]
Unfortunately @galli-leo closed this issue. I don't know if we should continue to discuss it here or create a new one.

**Comment by galli-leo** [comment]
Problem is, it doesn't really fit in any category. Backdrops aren't fanart by definition. They can be, but we don't know that.

**Comment by euphoria360** [comment]
@galli-leo There is no such thing as `<NameOfVideoFile>-backdrop.jpg/png` in [Kodi's Artwork Naming](https://kodi.wiki/view/Movie_artwork) and since the plugin supposed to be based on what Kodi needs, naming them like that would brake kodi.

**Comment by AnthraxLeprosyPi** [comment]
Maybe I'm blind, but what is the big deal with naming them -fanart and be done with it?
Imho backdrop is synonymous with fanart?

I bet there are bigger fish to fry ;-)

just my two cents,
Anthrax

**Comment by galli-leo** [comment]
Honestly, don’t know why I renamed the issue that way 😅 My brain must have shut off. Since I am on my phone only for a few days I cannot rename the issue. 
It’s not synonymous though, a backdrop can be provided eg by the studio

**Comment by AnthraxLeprosyPi** [comment]
I'm not sure if I can follow? What do you mean studio? 
In the context of a specific movie the backdrops are always fanrt or am I mistaken?

https://www.themoviedb.org/movie/284053-thor-ragnarok/images/backdrops

regards
Anthrax

**Comment by galli-leo** [comment]
Well in your example a lot of the backdrops are stills from the movie, so not fanart. Also for some backdrops, it seems more like official images than fanart, but it’s kinda hard to know.

**Comment by hikaricore** [comment]
None the less, these have always been considered to fall under the fanart category. TheMovieDB's preferred labelling shouldn't take precedence over the name of files that media applications actually use should it? As it stands one would have to run a post-processing script in addition to the metadata function to correct the names of these files, as nothing is going to be able to utilize them for their intended purpose with either the fully incorrect -banner or semantically incorrect -backdrop filename.  ^_^

**Comment by AnthraxLeprosyPi** [comment]
To be honest I feel like we're overthinking this... 

I really don't differentiate (or really care tbh) between fan provied artwort, studio provided wallpaper like artwork, movie stills, etc. What I like though is to have a nice 'backdrop' that is remotely related to the movie I'm browsing in Kodi - call me simple ;-)

high fives all around
Anthrax

**Comment by euphoria360** [comment]
Agreed.

**Comment by galli-leo** [comment]
I think you guys misunderstood my statement. I (or someone else) am gonna rename it to -fanart. I just thought maybe Kodi should also add -backdrop to the allowed endings, since it makes IMO more sense. Fanart isn't just backdrops, it could also be a poster, banner, etc.

**Comment by bobsaggetrfd** [comment]
Will this be fixed soon? Fanart is not being scanned into Kodi correctly because it's still named ~-banner.jpg

**Comment by galli-leo** [comment]
@bobsaggetrfd No idea. It took over a year to port this over from sonarr, so waiting a few weeks wont be too bad.

**Comment by jeremylee7** [comment]
Any update/eta on when the update with this fix will be, and will radarr scan files and fix the ones it named as -backdrop?

**Comment by galli-leo** [comment]
@jeremylee7 No certain eta yet, since an other pr needs to be merged first. Don't know about that, but they should be @Qstick

**Comment by Qstick** [comment]
@galli-leo, this doesn’t fix already created files as it doesn’t know which ones radarr created and which were actually user content. It will create new correct fanart versions though.
